### PR TITLE
Add "Terminology Changes" article & update 1.5 Release Notes

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Add a separated "Terminology Changes" article which details what we have changed, and why.

https://user-images.githubusercontent.com/99246719/225624981-15258119-5197-400d-bd14-b7de354bd90d.mov

- Update the Release Notes - focus on UX changes & mention Node-RED 3.1.0 Beta available

https://user-images.githubusercontent.com/99246719/225624793-25fb362c-ee86-4c28-ab32-15fe4118ef73.mov

## Related Issue(s)

#521 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: src/blog/2023/03/flowforge-1-5-0-released.md [hunk 1]
```diff
@@ -1,51 +1,66 @@
 ---
 title: FlowForge v1.5 Now Available
-subtitle: Updates to UI and architecture to allow for future features
-date: 2023-03-16 14:00:00.0
+subtitle: Updates to UI and architecture to allow for future features and Node-RED 3.1.0 Beta Available!
+description: For FlowForge 1.5 we have been busy making a lot of UX changes and upgrading our underlying architecture to enable future innovations on the FlowForge platform.
+date: 2023-03-16 15:00:00.0
 authors: ["ian-skerrett"]
```
Comment (iskerrett) [inline src/blog/2023/03/flowforge-1-5-0-released.md:6]:
```suggestion
authors: ["joe-pavitt"]
```

**Review by iskerrett — COMMENTED** [review]
This seems fine but I am really out of the loop on how we are suppose to be talking about things so I probably shouldn't be approving.

File: src/blog/2023/03/flowforge-1-5-0-released.md [hunk 2]
```diff
@@ -1,51 +1,66 @@
...
 
-We hope these changes will simplify the FlowForge terminology for our users and allow us to grow the FlowForge platform.
 
+![Screenshot to show an Instance's "Devices" view](./images/screenshot-devices.png)
+<figcaption class="-mt-6 text-center"><b>"Devices" view, available for a given Node-RED Instance. This lists all of the connected devices to a given instance, that will automatically update when a new Target Snapshot is set.</b></figcaption>
+
+
+## Node-RED 3.1 Beta Available
+
+FlowForge Cloud is a great place to try out the new Node-RED features, with FlowForge Cloud now including the [Node-RED 3.1.0-beta.2](https://discourse.nodered.org/t/node-red-3-1-0-beta-2-released/76192).
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/flowforge-1-5-0-released.md:52]:
```suggestion
FlowForge Cloud is a great place to try out the new Node-RED features, with FlowForge Cloud now including the [Node-RED 3.1.0-beta.2](https://discourse.nodered.org/t/node-red-3-1-0-beta-2-released/76192). If you want to try this version you can [duplicate your application](https://flowforge.com/docs/user/project-settings/#copy-project) or [upgrade your stack](https://flowforge.com/docs/user/changestack/).
```

File: src/blog/2023/03/flowforge-1-5-0-released.md [hunk 3]
```diff
@@ -1,51 +1,66 @@
...
 
 ## Other Improvements
 
-Update to audit logs to improve usability  [[#1800](https://github.com/flowforge/flowforge/issues/1800)] [[#1785](https://github.com/flowforge/flowforge/issues/1785)]
+- Update to audit logs to improve usability  [[#1800](https://github.com/flowforge/flowforge/issues/1800)] [[#1785](https://github.com/flowforge/flowforge/issues/1785)]
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/flowforge-1-5-0-released.md:56]:
```suggestion
- Update to audit logs to improve usability [[#1800](https://github.com/flowforge/flowforge/issues/1800)] [[#1785](https://github.com/flowforge/flowforge/issues/1785)]
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**